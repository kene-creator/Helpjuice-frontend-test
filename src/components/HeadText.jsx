const HeadText = () => {
  return (
    <div className="mt-6">
      <h1 className="text-[2.5rem] font-bold text-left flex justify-center items-center w-[50%] mx-auto border-b border-[#a4aab4]">
        Front-end developer test project
      </h1>
      <p className="text-[1rem] text-[#65676a] text-left mx-auto w-[50%] mt-5">
        The goal is to make a page that looks exactly like this one and has the
        ability to create H1 text simply by typing / then 1, then typing text,
        and hitting enter
      </p>
    </div>
  );
};

export default HeadText;
