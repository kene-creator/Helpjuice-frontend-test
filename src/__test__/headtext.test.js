import React from 'react';
import { render } from '@testing-library/react';
import HeadText from '../components/HeadText';

describe('HeadText component', () => {
  it('should render correctly', () => {
    const wrapper = render(<HeadText />);
    expect(wrapper).toMatchSnapshot();
  });
});
