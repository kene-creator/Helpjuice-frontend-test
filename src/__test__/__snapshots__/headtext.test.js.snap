// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeadText component should render correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="mt-6"
      >
        <h1
          class="text-[2.5rem] font-bold text-left flex justify-center items-center w-[50%] mx-auto border-b border-[#a4aab4]"
        >
          Front-end developer test project
        </h1>
        <p
          class="text-[1rem] text-[#65676a] text-left mx-auto w-[50%] mt-5"
        >
          The goal is to make a page that looks exactly like this one and has the ability to create H1 text simply by typing / then 1, then typing text, and hitting enter
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="mt-6"
    >
      <h1
        class="text-[2.5rem] font-bold text-left flex justify-center items-center w-[50%] mx-auto border-b border-[#a4aab4]"
      >
        Front-end developer test project
      </h1>
      <p
        class="text-[1rem] text-[#65676a] text-left mx-auto w-[50%] mt-5"
      >
        The goal is to make a page that looks exactly like this one and has the ability to create H1 text simply by typing / then 1, then typing text, and hitting enter
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
